fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1253,13 +1248,13 @@ revokePrivilegeStatement
     ;
 
 grantRoleStatement
-    : GRANT identifierOrString TO user                          #grantRoleToUser
-    | GRANT identifierOrString TO ROLE identifierOrString       #grantRoleToRole
+    : GRANT identifierOrStringList TO user                                                              #grantRoleToUser
```

Comment:
```suggestion
    : GRANT identifierOrStringList TO user                                                              #grantRoleToUser
    : GRANT identifierOrStringList TO USER user                                                              #grantRoleToUser
```
In the design doc, starrocks support to `grant role to USRE user_name` explicitly.

Reply:
This PR is only to support the show grants statement. The current RBAC needs to be improved in many details. I will split it into multiple PRs and submit them separately. Each PR should be controlled on one function as much as possible, which is more convenient for review code.